week-2/F-logical-operators/exercise.js

```diff
@@ -11,14 +11,14 @@ var cssLevel = 4;
 
 // Finish the statements below to check whether HTML, CSS knowledge are above 5
 // (hint: use the comparison operator from before)
-var htmlLevelAbove5;
-var cssLevelAbove5;
+var htmlLevelAbove5 = htmlLevel > 5;
+var cssLevelAbove5 = cssLevel > 5;
 
 // Finish the next two statements
 // Use the previous variables and logical operators
-// Do not "hardcode" the answers
-var cssAndHtmlAbove5;
-var cssOrHtmlAbove5;
+// Do not "hardcode"  answers
+var cssAndHtmlAbove5 = htmlLevel > 5 && cssLevel > 5;
```

Comment:
You're repeating your work from earlier here. Try `htmlLevelAbove5 && cssLevelAbove5` instead as you've already got the values for these.

Reply:
The answer is correct but you are duplicating work already done.